### PR TITLE
Update URL of "Arducam_Mega"

### PR DESCRIPTION
**Pull request by ArduCAM**

### CODE REVIEW

**Comment by per1234** [comment]
Hello @ArduCAM. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by umbynos — APPROVED** [review]

**Comment by umbynos** [comment]
Done in https://github.com/arduino/library-registry/pull/2174#issuecomment-1346869603

**Pull request merged**
